server/src/test/java/io/crate/integrationtests/SQLTypeMappingTest.java

```diff
@@ -184,6 +183,8 @@ public void testInvalidInsertIntoObject() throws Exception {
             .hasPGError(INTERNAL_ERROR)
             .hasHTTPError(BAD_REQUEST, 4000)
             .hasMessageContaining("Cannot cast object element `created` with value `true` to type `timestamp with time zone`");
+
+        waitNoPendingTasksOnAll();
```

Comment:
What will this help here? Either the assertion will kick in or the test will fail anyhow